cmd/apps/influxdb_app.go

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) arkade author(s) 2021. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package apps
+
+import (
+	"strconv"
+
+	"github.com/alexellis/arkade/pkg"
+	"github.com/alexellis/arkade/pkg/apps"
+	"github.com/alexellis/arkade/pkg/config"
+	"github.com/alexellis/arkade/pkg/types"
+	"github.com/spf13/cobra"
+)
+
+func MakeInstallinfluxdb() *cobra.Command {
+	var influxdbApp = &cobra.Command{
+		Use:          "influxdb",
+		Short:        "Install influxdb",
+		Long:         "Install Influxdb into your cluster",
+		Example:      "arkade install influxdb --persistence",
+		SilenceUsage: true,
+	}
+
+	influxdbApp.Flags().StringP("namespace", "n", "default", "The namespace to install chartmuseum (default: default")
+	influxdbApp.Flags().Bool("update-repo", true, "Update the helm repo")
+	influxdbApp.Flags().Bool("persistence", false, "Enable persistence for influxdb (default: false)")
+
+	influxdbApp.RunE = func(command *cobra.Command, args []string) error {
+		kubeConfigPath, _ := command.Flags().GetString("kubeconfig")
```

Comment:
Do we need `kubeconfig` here? I thought we moved it up higher for each command.

Reply:
we need to grab it from here, but its defined at top level up